examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -39,7 +39,7 @@ case class UnsubscribeReceiver(receiverActor: ActorRef)
 class FeederActor extends Actor {
 
   val rand = new Random()
-  var receivers: LinkedList[ActorRef] = new LinkedList[ActorRef]()
+  val receivers = new ArrayBuffer[ActorRef]()
```

Comment:
Would a mutable Set be a better fit here? It seems that `receivers` is used to hold distinct ActorRefs